pom.xml

```diff
@@ -52,7 +52,7 @@
     <cxf.karaf.patched.version>3.0.13-pentaho</cxf.karaf.patched.version>
     <pentaho-camel.version>9.0.0.0-SNAPSHOT</pentaho-camel.version>
     <pdi-monitoring-plugin.version>9.0.0.0-SNAPSHOT</pdi-monitoring-plugin.version>
-    <com.fasterxml.jackson.core.version>2.9.4</com.fasterxml.jackson.core.version>
+    <fasterxml-jackson.version>2.9.5</fasterxml-jackson.version>
```

Comment:
This is already declared in the parent POM, no need to declare it here.
https://github.com/pentaho/maven-parent-poms/pull/66/files#diff-600376dffeb79835ede4a0b285078036R101